docs/app/documentation/component-docs/tabs/examples/tabs-navigation-mode-example.component.html

```diff
@@ -0,0 +1,30 @@
+<fd-tab-list>
```

Comment:
Is there a reason we need fd-tab-list as a wrapper component?

Reply:
There was a reason why we needed it, because of keyboard functionality, that was provided by wrapper. Right now It's changed and external service keeps this functionality, so wrapping is not needed anymore. :)